src/pages/api/citations/[...filename].page.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import path from 'path';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getManuscript } from '../../../manuscripts';
+import { config } from '../../../config';
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  const { filename } = req.query;
+  if (filename !== undefined) {
+    const { name, ext } = path.parse(Array.isArray(filename) ? filename[0] : filename);
+
+    const manuscript = getManuscript(config.manuscriptConfigFile, name);
+
+    if (manuscript !== undefined && name !== undefined && (ext === '.bib' || ext === '.ris')) {
+      const extReq = await axios.get(
+        `${config.apiServer}/api/citations/${manuscript.preprintDoi}/${ext === '.bib' ? 'bibtex' : 'ris'}`,
```

Comment:
I'm pretty certain this needs to be a constructed DOI for the RPP, not the preprint's DOI

Reply:
Good spot, I've lifted the logic that gets the eLife DOI when we fetch data from server and re-used it

Reply:
Yup tested on the latest preview env, works like a charm!